src/components/common/FormikUIAutocomplete.tsx

```diff
@@ -0,0 +1,62 @@
+import { InputProps } from '@mui/material/Input';
+import MuiTextField, { TextFieldProps } from '@mui/material/TextField';
+import { Field } from 'formik';
+import { Autocomplete } from 'formik-mui';
+import React from 'react';
+
+import { Option } from 'utils/utilTypes';
+
+type FormikUIAutocompleteProps = {
+  items: Option[];
+  name: string;
+  label: string;
+  loading?: boolean;
+  noOptionsText?: string;
+  required?: boolean;
+  disabled?: boolean;
+  InputProps?: Partial<InputProps> & { 'data-cy': string };
+  'data-cy'?: string;
+};
+
+const FormikUIAutocomplete: React.FC<FormikUIAutocompleteProps> = ({
+  items,
+  name,
+  label,
+  loading = false,
+  noOptionsText,
+  required,
+  disabled,
+  InputProps,
+  ...props
+}) => {
+  const options = items.map((item) => item.value);
+
+  return (
+    <Field
+      id={name + '-input'}
+      name={name}
+      component={Autocomplete}
+      loading={loading}
+      options={options}
+      noOptionsText={noOptionsText}
+      getOptionLabel={(option: number | string) => {
```

Comment:
I think it would make things little cleaner if the method `getOptionLabel` is injected as a prop from the place the component `FormikUIAutocomplete` is used.
This would help f.x. in CallGeneralInfo, where instead of creating `proposalWorkflowOptions` array to be used as items, the items would be simply the property `proposalWorkflows` itself, and there would be just a function that get's the label from that array.

This however could be done in a separate PR, but just to keep in mind

Reply:
I will create separate task for this and it could be revisited a bit better. Maybe we find some other things like this to expose for improved reusability.